integrationTests/vm/wasm/wasmvm/async_mock_contracts_test.go

```diff
@@ -165,3 +165,89 @@ func testMockContract_CrossShard(t *testing.T, asyncCallType []byte) {
 	require.Nil(t, err)
 	require.Equal(t, ownerOfParent.Address, originalCallerChild)
 }
+
```

Comment:
maybe rename the file to asyncMockContracts_test.go ? We do not regularly use snake case